### PR TITLE
Fix CSI shcd code to work with new paddle file format

### PR DESCRIPTION
**Pull request by thiagonache**
#### Summary and Scope

- Relates to CASMINST-3649

##### Issue Type

- Bugfix Pull Request

This Pull Requests fixes csi config shcd command to generate CMS Installation files.

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

File: internal/files/shcd-schema.json [hunk 1]
```diff
@@ -1,87 +1,82 @@
 {
```
Comment (jacobsalmela) [inline internal/files/shcd-schema.json:1]: This doesn't seem to match the paddle schema from canu: https://github.com/Cray-HPE/canu/blob/develop/network_modeling/schema/paddle-schema.json

**Review by jacobsalmela — CHANGES_REQUESTED** [review]

File: internal/files/shcd-schema.json (hunk 1 above)
Comment (thiagonache) [reply internal/files/shcd-schema.json:1]: fixed

Comment (jacobsalmela) [reply internal/files/shcd-schema.json:1]: I still see some differences between the two.  Can you just copy the file over directly so they match?  Or do a git submodule to keep it in sync, or whatever method works.

Comment (thiagonache) [reply internal/files/shcd-schema.json:1]: fixed in canu project and they match 100% now.

**Review by jacobsalmela — APPROVED** [review]

**Pull request merged**